### PR TITLE
📖 Update Mediavine docs

### PR DESCRIPTION
**Pull request by timtyrrell**
Detail how the `consentHandlingOverride` config value changes ad personalization

### CODE REVIEW

**Comment by timtyrrell** [comment]
@zhouyx I believe this is the doc update that I owed you, please let me know if you meant something else, thanks

File: ads/mediavine.md [hunk 1]
```diff
@@ -34,3 +34,13 @@ For details on the configuration semantics, please contact [Mediavine](http://ww
 * `data-site` - The site's unique name this ad will be served on. This is the same name from your Mediavine script wrapper.
 
 Each site must be approved and signed up with [Mediavine](http://www.mediavine.com) prior to launch. The site name will be the same as name in the Mediavine script wrapper. The site name `amp-project` can be used for testing and will serve placeholder ads.
+
+## Consent and GDPR
+ 
+Mediavine has set the `consentHandlingOverride` config value to `true` meaning that EU traffic will be prompted with the `<amp-consent>` component. Ad personalization is affected based on the user's interaction with this component.
```
Comment (zhouyx) [inline ads/mediavine.md]: This is not correct. User will still need to include the `data-block-on-consent` attribute and `<amp-consent>` component to enable to this feature. 

@lannka I think we should come up with a template documentation here. WDYT

Comment (lannka) [reply ads/mediavine.md]: Agree, we can have a place to have a unified language for this. `amp-ad.md` might be a good place.

Comment (lannka) [reply ads/mediavine.md]: sorry, it should be ads/README.md, given it's ad network facing documentation.

Comment (zhouyx) [reply ads/mediavine.md]: or we can have create a documentation for `_ping_`, and refer to `_ping_.md`

Comment (timtyrrell) [reply ads/mediavine.md]: Should I wait for the maintainers to make a template before I make this doc change?

Comment (lannka) [reply ads/mediavine.md]: @timtyrrell FYI, the template is there: https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md#user-consent-integration

**Comment by timtyrrell** [comment]
New PR here, I tried to match up with the sample docs, let me know otherwise, thanks: https://github.com/ampproject/amphtml/pull/16757

**Pull request closed without merging**